strapi/strapi
# S3 File Upload larger than 5MB invalid file url

### What does it do?

When upload file more than 5MB with AWS S3 provider it returns file location without bucket and endpoint, it returns only name of the file starts with slash. `ex: /img_hash456.png`.

Create condition that check if data.location starts with "/", if starts add protocol, bucket and endpoint before file name and return as file url.

 - packages\providers\upload-aws-s3\src\index.ts lines 99 - 101

### Why is it needed?

Invalid url when file is larger than 5MB. 

### How to test it?

check test upload-aws-s3.test.ts - Should add to the url the https protocol, bucket and endpoint if it is missing

### Related issue(s)/PR(s)

issue: https://github.com/strapi/strapi/issues/15144
